js/index.js

```diff
@@ -1,7 +1,47 @@
 // Iteration 1: Names and Input
+const hacker1 = 'Nidian'
+console.log(`"The driver's name is ${hacker1}"`)
 
+const hacker2 = 'Maria'
+console.log(`"The navigator's name is ${hacker2}"`)
```

Comment:
Cuando usamos ES6 y los back-ticks (``), no hace falta incluir comillas en el string "" o '', podríamos en estos casos y en los siguientes no incluirlas, quedarían algo asi:

`console.log(`**The navigator's name is ${hacker2}**`)`